docs/how-to-provision-using-dps.md

```diff
@@ -413,52 +437,69 @@ namespace Samples.AdtIothub
+Now you need to set up an IoT Hub route, to route device lifecycle events. In this case, you will specifically listen to device delete events, identified by `if (opType == "deleteDeviceIdentity")`. This will trigger the delete of the digital twin item, finalizing the retirement of a device and its digital twin.
+
+Instructions for creating an IoT Hub route are described in this article: [*Use IoT Hub message routing to send device-to-cloud messages to different endpoints*](../iot-hub/iot-hub-devguide-messages-d2c). The section *Non-telemetry events* explains that you can use **device lifecycle events** as the data source for the route.
 
 The steps you need to go through for this setup are:
-- Create a custom IoT Hub event hub endpoint. This endpoint should target the event hub you create in the 'Create an Event Hub' section.
-- Add a Device Lifecycle Events route. Use the endpoint created in the previous step. You can limit the device lifecycle events to only send the delete events by adding the routing query `opType='deleteDeviceIdentity'`.
+* Create a custom IoT Hub event hub endpoint. This endpoint should target the event hub you create in the 'Create an Event Hub' section.
+* Add a Device Lifecycle Events route. Use the endpoint created in the previous step. You can limit the device lifecycle events to only send the delete events by adding the routing query `opType='deleteDeviceIdentity'`.
     ![Add a route](media/lifecycle-route.png)
 
-Once you gone through these steps everything is set to retire devices end-to-end.
+Once you have gone through these steps, everything is set to retire devices end-to-end.
 
 ### Validate
 
-To trigger the process of retirement you need to manually delete the device from IoT Hub. Use the process described in part one of this article to create a device and Twin instance. Then go to the IoT Hub and delete the device. The device will be automatically removed from Azure Digital Twins. Use the following command to verify the Twin of the device in the Azure Digital Twins instance was deleted.
+To trigger the process of retirement, you need to manually delete the device from IoT Hub.
+
+Use the process described in [part one](#auto-provision-device-using-device-provisioning-service) of this article to create a device and corresponding digital twin. 
+
+Then, go to the IoT Hub and delete the device. The device will be automatically removed from Azure Digital Twins. 
+
+Use the following command to verify the Twin of the device in the Azure Digital Twins instance was deleted.
 
 ```azurecli-interactive
 az dt twin show -n <Digital Twins instance name> --twin-id <Device Registration ID>"
 ```
 
 The output should tell you the twin-id doesn't exist.
 
+## Mapping ID values
```

Comment:
I moved this content out of Next Steps because it didn't seem like a next step.

However, you may want to retitle this section based on what the intent was for including it.

Reply:
Agree. Will retitle and rewrite a bit.